JWBestPracticeApps/FeedTableViewController/FeedTableViewController.swift

```diff
@@ -33,20 +34,25 @@ class FeedTableViewController: UITableViewController {
                 continue
             }
 
-            if let player = buildPlayer(title: title, url: url) {
+            if let player = buildPlayer(title: title, url: url, thumbnail: itemInfo["thumbnail"]) {
                 feed.append(player)
             }
+            //Populate the thumbnailsIdentifiers array with local thumbnail name
+            if let localThumbnail = itemInfo["localThumbnail"] {
+                thumbnailsIdentifiers[title] = localThumbnail
+            }
         }
     }
 
-    open func buildPlayer(title: String, url: String) -> JWPlayerController? {
+    open func buildPlayer(title: String, url: String, thumbnail: String?) -> JWPlayerController? {
```

Comment:
Thumbnails aren't used in this project. I would exclude it from this project. Instead what you can do is have `buildPlayer(title: String, url: String, thumbnail: String?)` from the Autoplay project call `buildPlayer(title: String, url: String)` internally. The FeedTableViewController should be kept  as simple as possible.

Reply:
I thought about doing something like that, the problem I encountered is that `FeedTableViewController` is using `fileprivate func fetchFeed()` to fetch the videos from the Feed.plist and also populates feed array with `buildPlayer(title: String, url: String)`.
So to use `buildPlayer(title: String, url: String, thumbnail: String?)` in `AutoplayVideoFeedViewController` I will have also to replicate the `fetchFeed()` and add a Feed.plis to it and then I will no longer make sense to inherit from `FeedTableViewController`

Reply:
@fernandoavalos What if you change the scope of `fetchFeed()` to be `internal` instead of `fileprivate`?

Reply:
@s-seibert that could work, that way I could also do my own implementation of `fetchFeed()`. By doing this I will also have to add a Feed.plist to `AutoplayVideoFeedViewController` Do you guys agree that I should do this modification like this? @karimMourra

Reply:
why do you need to make a new Feed.plist? Why not continue to use the one from FeedTableViewController? That's what Autoplay is currently using.

Reply:
@fernandoavalos , as @s-seibert said, i don't think you need a copy of Feed.plist. You can make sure it is accessible by other projects in the workspace instead.